chef/cookbooks/neutron/recipes/post_install_conf.rb

```diff
@@ -84,7 +84,11 @@ def mask_to_bits(mask)
     end
   when 'gre'
     fixed_network_type = "--provider:network_type gre --provider:segmentation_id 1"
-    floating_network_type = "--provider:network_type gre --provider:segmentation_id 2"
+    if node[:network][:networks][:nova_floating][:use_vlan]
```

Comment:
> yeah, I was wondering about that- I'm not sure about the vxlan case (see below).

For provider networks, I think only `flat` and `vlan` are useful. From http://docs.openstack.org/admin-guide-cloud/content/tenant-provider-networks.html : 
```
Provider networks. Provider networks are created by the OpenStack administrator and map directly to an existing physical network in the data center. Useful network types in this category are flat (untagged) and VLAN (802.1Q tagged). It is possible to allow provider networks to be shared among tenants as part of the network creation process.
```